ndex2/client.py

```diff
@@ -482,9 +594,31 @@ def get_neighborhood_as_cx_stream(self, network_id, search_string, search_depth=
 
     def get_neighborhood(self, network_id, search_string, search_depth=1, edge_limit=2500):
         """
+
         Get the CX for a subnetwork of the network specified by UUID network_id and a traversal of search_depth steps
         around the nodes found by search_string.
 
+        .. code-block:: python
+
+		import ndex2.client
+		my_account="your account"
+		my_password="your password"
+		my_ndex=ndex2.client.Ndex2("http://public.ndexbio.org", my_account, my_password)
+
+
+		# Query the network from my NDEx account, get small subset
+		query_result_cx=my_ndex.get_neighborhood('INSERT NETWORK UUID HERE','INSERT NODE NAME HERE')
```

Comment:
Instead of making the user find a network they can run a neighborhood query on just directly create a network with this fragment of code (you'll have to move the import to the top.

```
from ndex2.nice_cx_network import NiceCXNetwork

my_net = NiceCXNetwork()
my_net.create_node('testnode1')
my_net.set_name('testnetwork')
uri_subset = my_ndex.save_new_network(my_net.to_cx())
```